src/index.html

```diff
@@ -1,10 +1,33 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <script src="./requirements.js"></script>
-    <title></title>
-  </head>
-  <body>
-  </body>
+    <head>
+        <meta charset="utf-8">
+        <script type="module" src="./requirements.js"></script>
+        <link rel="stylesheet" type="text/css" href="gui/style/style.css">
+        <title></title>
+    </head>
+    <body>
+        <div id="editor-header">
```

Comment:
You should probably not directly add your view to the body. Wrap it in a div and name it the name of whatever view. I.E. when vatricia's login get's added it will be in a div called login and we can show that by switching whether the id="login" shows or the id="student-view" etc shows.